wgpu-hal/src/gles/wgl.rs

```diff
@@ -127,7 +127,7 @@ impl WglContext {
         if unsafe { OpenGL::wglGetCurrentContext() }.is_invalid() {
             return Ok(());
         }
-        unsafe { OpenGL::wglMakeCurrent(None, None) }
+        unsafe { OpenGL::wglMakeCurrent(Default::default(), Default::default()) }
```

Comment:
@kennykerr can you help us here? I did not do my due diligence this time to search through `windows-rs` and `win32metadata` for relevant changes, but while most handles have been made optional (see other `Some()` wrapping changes in this PR), quite strangely these parameters are no longer optional in `wglMakeCurrent()` where they have a specific use-case to uncurrent the context.

Is this change intended or accidental? We can clearly get around it by passing a defaulted `HDC`/`HGLRC`, but it seems strange when compared to other functions.

Reply:
The code generator now faithfully honors the metadata's `[Optional]` attribute - in the case of `wglMakeCurrent` the handle parameters are not optional whereas in the case of `CreateSwapChainForCompositionSurfaceHandle` the handle parameter is optional.

Reply:
Thanks @kennykerr. So this'll take another trip through `win32metadata` to annotate correctly.